schemas/VERSION

```diff
@@ -1 +1 @@
-2023.10.2
+2023.10.3
```

Comment:
Is this version bump just so that we test that it still works when it merges to main?

Reply:
Yeah I had to trigger a deploy to test it out.